metadata-db-app/src/cmr/metadata_db/services/concept_service.clj

```diff
@@ -776,15 +777,29 @@
       ;; to send concept updates and deletions out of order.
       (if (and (util/is-tombstone? previous-revision) (nil? revision-id))
         previous-revision
-        (let [metadata (if (cu/generic-concept? concept-type)
+        (let [metadata (if (or (cu/generic-concept? concept-type)
+                               (= :subscription concept-type))
                          (:metadata previous-revision)
                          "")
-              tombstone (merge previous-revision {:concept-id concept-id
-                                                  :revision-id revision-id
-                                                  :revision-date revision-date
-                                                  :user-id user-id
-                                                  :metadata metadata
-                                                  :deleted true})]
+              tombstone (if (= :subscription concept-type)
```

Comment:
Does this update not need to be done for the other save-concept-revision? Is it only relevant for tombstones?